src/main/java/org/commcare/formplayer/application/UtilController.java

```diff
@@ -102,7 +106,12 @@ public NotificationMessage deleteApplicationDbs(
     @UserLock
     public NotificationMessage clearUserData(
             @RequestBody AuthenticatedRequestBean requestBean,
-            HttpServletRequest request) {
+            HttpServletRequest request) throws InvalidStructureException {
+
+        // Test clearing case db
```

Comment:
nit: can this comment be removed? Or is this still in the testing phase?

Reply:
i'll remove it